readme.md

```diff
@@ -25,8 +25,8 @@ ___
 
 ❗️ Replace `<your_account>` with your Github username and copy the links to `Pull Request` description:
 
-- [DEMO LINK](https://<your_account>.github.io/layout_search-bar-airbnb/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_search-bar-airbnb/report/html_report/)
+- [DEMO LINK](https://silchenko-arsen.github.io/layout_search-bar-airbnb/)
+- [TEST REPORT LINK](https://silchenko-arsen.github.io/layout_search-bar-airbnb/report/html_report/)
```

Comment:
Add these links to the PR description
<img width="955" alt="image" src="https://github.com/mate-academy/layout_search-bar-airbnb/assets/64750958/c92a8bac-5940-414f-b135-3bdafb4820aa">